modules/node-observability-install-web-console.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/understanding-node-observability-operator.adoc
+
+:_content-type: PROCEDURE
+[id="install-node-observability-using-web-console_{context}"]
+= Installing the Node Observability Operator using the web console
+
+You can install the Node Observability Operator from the {product-title} web console.
+
+.Prerequisites
+
+* You have access to the cluster with `cluster-admin` privileges.
+* You have access to the {product-title} web console.
+
+.Procedure
+
+. Log in to the {product-title} web console.
+. In the Administrator's navigation panel, expand *Operators* → *OperatorHub*.
+. In the *All items* field, enter *Node Observability Operator* and select the *Node Observability Operator* tile.
+. Click *Install*.
+. On the *Install Operator* page, configure the following settings:
+.. In the *Update channel* area, click *alpha*.
+.. In the *Installation mode* area, click *A specific namespace on the cluster*.
+.. From the *Installed Namespace* list, select *node-observability-operator* from the list.
+.. In the *Update approval* area, select *Automatic*.
+.. Click *Install*.
+
+.Verification
+. In the Administrator's navigation panel, expand *Operators* → *Installed Operators*.
```

Comment:
Same here.

Reply:
It's discussed in this thread -https://github.com/openshift/openshift-docs/pull/47075#discussion_r926467032.